resource/layout/settingssubstreaming.layout

```diff
@@ -90,18 +106,37 @@
 			spacing=10
 		}
 
-		place { control="Pair" start=RemoteClientList dir=down margin-top=10}
-		place { control="UnPair,SetPIN" start=Pair dir=right spacing=20 margin-left=20 }
+		place { control=Pair start=RemoteClientList dir=down margin-top=10}
+		place { control="UnPair,SetPIN" start=Pair dir=right spacing=20 margin-left=20 }	
+
+		place
+		{
+			start=Pair
+			control=P2PScopeLabel
+			dir=down
+			width=max
+			margin-top=20
+		}
+		place
+		{
+			start=P2PScopeLabel
+			control=P2PScope
+			dir=down
+			width=max
+			height=24
+			margin-right=20
+			margin-top=10
+		}
 		
-		place { region=bottom control=HostDivider width=max}
-		place { region=bottom control=HostLabel width=max start=HostDivider dir=down margin-top=15 }
+		place { region=bottom control=HostDivider start=P2PScope width=max dir=down margin-top=15}
+		place { region=bottom control=HostLabel width=max start=HostDivider dir=down margin-top=10 }
 		place { region=bottom control=UnsupportedPlatform width=max start=HostLabel dir=down margin-top=10 }
 		place { region=bottom control=AdvancedHostOptions start=HostLabel dir=down margin-top=10 }
 		
-		place { region=bottom control=ClientDivider width=max start=AdvancedHostOptions dir=down margin-top=15 }
-		place { region=bottom control=ClientLabel width=max start=ClientDivider dir=down margin-top=15 }
-		
-		place { region=bottom control=QualityFast start=ClientLabel dir=down margin-top=10}
+		place { region=bottom control=ClientDivider width=max start=AdvancedHostOptions dir=down margin-top=15}
+		place { region=bottom control=ClientLabelLocal width=max start=ClientDivider dir=down margin-top=10 }
+		place { region=bottom control=ClientLabelRemote width=max start="ClientDivider" dir=down margin-top=25 }
```

Comment:
Since ClientLabelLocal & ClientLabelRemote are interchangeable depending on context they need the same styles
```suggestion
		place { region=bottom control=ClientLabelRemote width=max start="ClientDivider" dir=down margin-top=10 }
```